### DISCUSSION TITLE
ike-updown event behavior on load-conn and initiate conn command

### DISCUSSION THREAD

**Discussion by hbhokare** [strongswan/strongswan#562] [Q&A]
Hi,
We are trying to develop the IPSec-based application, using different strongswan/davici API from our application. We have below questions related to ike-updown event behavior on load-conn and initiate conn command 

1. we can see ike-updown event ( without up flag ) which means ike-down event even before we do first initiate conn for given conn. In this case, we have config loaded in strongswan using the load conn command on both sites. Is it correct behavior? Can we see ike-down event for given conn without getting ike-up event first ( or when strongswan is waiting for initiate conn command from the caller )

**Comment by tobiasbrunner** [comment]
>  we can see ike-updown event ( without up flag ) which means ike-down event even before we do first initiate conn for given conn.

Read the logs to see what's going on.
